base_layer/wallet_ffi/tests/fixtures/mock_callbacks.py

```
@@ -0,0 +1,416 @@
+                return [inv for inv in self.invocations if inv.callback_name == name]
+    
+    # Mock callback implementations
+    
+    def _mock_received_transaction(self, tx_data: Dict[str, Any]):
+        """Mock implementation of received_transaction callback."""
+        print(f"[MOCK] Transaction received: ID={tx_data.get('tx_id', 'unknown')}, "
+              f"Amount={tx_data.get('amount', 0)} µT")
+        
+        # Validate expected fields
+        expected_fields = ['tx_id', 'source_address', 'amount', 'fee', 'message', 'timestamp']
+        for field in expected_fields:
```

Flake8 (7.2.0)

[error] import 'field' from line 13 shadowed by loop variable

(F402)